backend/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend
+
+go 1.12
```

Comment:
is this needed?

Reply:
Those files are automatically generated by go commands